crates/turborepo-lib/src/run/scope/change_detector.rs

```diff
@@ -52,13 +52,12 @@ impl<'a> ScopeChangeDetector<'a> {
     }
 
     /// Gets the lockfile content from SCM if it has changed.
-    /// Does *not* error if cannot get content, instead just
-    /// returns a Some(None)
-    fn get_lockfile_contents(
+    /// Does *not* error if cannot get content.
+    pub fn get_lockfile_contents(
         &self,
         from_ref: Option<&str>,
         changed_files: &HashSet<AnchoredSystemPathBuf>,
-    ) -> Option<Option<Vec<u8>>> {
```

Comment:
We're going from 3 states to 4 states with the return type change, but we never return `LockfileContents::Unknown` from this function. Is that variant needed?